examples/jenkins/pipeline/pipelinetemplate.json

```diff
@@ -19,6 +19,9 @@
         "creationTimestamp": null,
         "labels": {
           "name": "sample-pipeline"
+        },
+        "annotations": {
+          "downstream.openshift.io/deployments": "[{\"name\": \"frontend\", \"namespace\": \"\"}]"
```

Comment:
that's a question for @spadgett, i'm not sure exactly what the UI wants/needs to show.  I think the original goal was to show things created by the pipeline, not necessarily things "used by" the pipeline.  so if the pipeline is creating a route, doing some tests, and tearing it down, then yes... if not, then no, would be my guess.